feature/client-api/src/main/java/com/simprints/feature/clientapi/mappers/request/validators/ConfirmIdentityValidator.kt

```diff
@@ -37,9 +40,40 @@ internal class ConfirmIdentityValidator(
         }
     }
 
-    private fun validateSelectedGuid(selectedId: String) {
+    private suspend fun validateSelectedGuid(
+        selectedId: String,
+        sessionId: String,
+    ) {
         if (selectedId.isBlank()) {
             throw InvalidRequestException("Missing Selected GUID", ClientApiError.INVALID_SELECTED_ID)
         }
+
+        // Allow 'NONE_SELECTED' as a special case to indicate no selection
+        if (selectedId.equals("NONE_SELECTED", ignoreCase = true)) {
```

Comment:
I remembered that we have this value defined in the LibSimprints - https://github.com/Simprints/LibSimprints/blob/ddc3b719a514b01fcf686547e6cac61f6dd1c537/src/main/java/com/simprints/libsimprints/Constants.kt#L27